scripts/_job.py

```diff
@@ -63,5 +63,8 @@ def run_job(steps: List[Step]) -> None:
         end = time.perf_counter()
         logger.success(f"End: '{step.name}', runtime: {end - start:.3f} seconds.")
 
+    if not failed:
+        __rollback_job(rollback_steps)
```

Comment:
Unrelated: run the rollbacks when none of the steps failed (instead of having to specify the rollback as a step)